imageprocessor/templates/base.html

```diff
@@ -2,16 +2,44 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+	{% load staticfiles %}
+	<link rel='stylesheet' href="{% static 'css/bootstrap.min.css' %}" type='text/css'/>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="icon" href="{% static 'img/favicon.ico' %}">
     <title>Photo Tagger</title>
+	
     <script src="//ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>
+	<script src="{% static 'js/bootstrap.min.js' %}"></script>
     {% cloudinary_includes %}
     {% cloudinary_js_config %}
      {% block css %}{% endblock %}
     <script>
       {% block javascript %}{% endblock %}
     </script>
   </head>
-  <body>
+  <body style="background-color:#f6f6f6">
+  <header>
+		  <nav class="navbar navbar-expand-lg navbar-light bg-light" style="height: 60px;">
+		  <a class="navbar-brand" href="/">
+			<img src="{% static 'img/logo.png' %}" width="100" height="100" alt="Photo Tagger">
+		  </a>
+		  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+			<span class="navbar-toggler-icon"></span>
+		  </button>
+		  <div class="collapse navbar-collapse" id="navbarNav">
+			<ul class="navbar-nav">
+			  <li class="nav-item active">
+				<a class="nav-link" href="/">Home <span class="sr-only">(current)</span></a>
```

Comment:
Since you have a logo that links to the home (root) page, you don't need a second link called home that also links to  the homepage

Reply:
removed home button